iktomi/cli/app.py

```diff
@@ -143,7 +143,7 @@ def iter_module_files():
     for module in sys.modules.values():
         filename = getattr(module, '__file__', None)
         if filename:
-            while not os.path.isfile(filename):
+            while not os.path.isfile(filename): # pragma: no cover
```

Comment:
This loop doesn't seem to do something meaningful. Please check the source whre it was copied from.

Also the line `if filename[-4:] in ('.pyc', '.pyo'):` below can be expressed in better way as `if filename.endswith(('.pyc', '.pyo')):`.